scripts/codegen_raw_models.py

```
@@ -0,0 +1,642 @@
+        GENERATED_HEADER
+        + '"""Permissive view models for parsing real-world Kibana dashboard JSON."""\n\n'
+        + 'from kb_dashboard_tools.decompile.kbn_raw_models.dashboard.view import KbnDashboard\n\n'
+        + "__all__ = ['KbnDashboard']\n"
+    )
+
+    print(f'\nGenerated {len(generated)} files in {OUT_DIR.relative_to(REPO_ROOT)}')
+
+    # Auto-format with ruff to ensure CI passes.
+    tools_dir = REPO_ROOT / 'packages' / 'kb-dashboard-tools'
+
+    ruff_check = subprocess.run(
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)